operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -10,6 +10,9 @@
 # This is useful when using `helm template`, because then helm will use the client version of kubectl as the Kubernetes version,
 # which may or may not match your cluster's server version. Example: 'v1.24.4'. Set to null to use the version that helm
 # devises.
+
+# Note: The values in this file is not backward compatible. If you copy and paste the values for your installation configuration,
+# make sure to use the values.yaml from branch or tag that matches the mimir-distributed helm chart version you want to install.
```

Comment:
I would discourage people would copy pasting the whole thing like in the getting started guide to be consistent

Reply:
Rephrased the sentence, please review again.